khmer/thread_utils.py

```diff
@@ -181,9 +181,9 @@ def do_write(self, outfp):
             except Queue.Empty:
                 continue
 
-            for name, seq, accuracy in g.seqlist:
-                if accuracy:  # write FASTQ; CTB hack.
-                    outfp.write('@%s\n%s\n+\n%s\n' % (name, seq, accuracy))
+            for name, seq, quality in g.seqlist:
+                if quality:  # write FASTQ; CTB hack.
```

Comment:
On Tue, Feb 10, 2015 at 08:03:56AM -0800, bocajnotnef wrote:
> > @@ -181,9 +181,9 @@ def do_write(self, outfp):
> >              except Queue.Empty:
> >                  continue
> >  
> > -            for name, seq, accuracy in g.seqlist:
> > -                if accuracy:  # write FASTQ; CTB hack.
> > -                    outfp.write('@%s\n%s\n+\n%s\n' % (name, seq, accuracy))
> > +            for name, seq, quality in g.seqlist:
> > +                if quality:  # write FASTQ; CTB hack.
> 
> I can't quite figure out how to pass the individual records to write_record; I'm not sure I understand what the setlist is. I assume a list, yet 
> 
> ```for item in g.setlist:```
> ```    write_record(item, outfp)```
> 
> doesn't work.

ahh, it's a tuple - so you'd need to construct another version of
write_record that took a tuple.  So never mind, leave it as it is.
This code should go away soon anyway :)

--titus
-- 
C. Titus Brown, ctbrown@ucdavis.edu